main/src/test/scala/ParseKey.scala

```diff
@@ -137,5 +175,8 @@ object ParseKey extends Properties("Key parser test") {
     } yield structure
 
   def pickN[T](load: Double, from: Seq[T]): Gen[Seq[T]] =
-    pick((load * from.size).toInt, from)
+    pick((load * from.size).toInt match {
+      case 0 => 1
+      case x => x
```

Comment:
`pickN` is a function that randomly picks `load` % items from the `from` sequence, but the rest of the tests expect at least one item, so I changed it to return 1 in case of 0. In grand scheme of things, it's a random build generator for tests, and I don't think it compromises the coverage of the test.